dotnet-client-libraries/WireMock.NET/WireMockNet/Planet.cs

```diff
@@ -1,25 +1,11 @@
 ﻿namespace WireMockNet
 {
-    public class Planet
+    public record class Planet
     {
         public int Id { get; set; }
```

Comment:
This should either be {get; init;} or be switched to the primary constructor type syntax. Records are a special type of readonly and are not supposed to support set on members. There is the special with operator syntax for constructing a copy with modified values. I guess I have never tested, but I am surprised the compiler didn't give a warning on this.

Reply:
I just checked the docs, and sure enough, you can create modifiable properties on a record, but by default they are init only. Thanks for making me check the docs 😀

Reply:
I like the primary constructor apparently will switch to this. Thanks for the advices